firefox-ios/firefox-ios-tests/Tests/XCUITests/NavigationTest.swift

```diff
@@ -403,7 +413,7 @@ class NavigationTest: BaseTestCase {
         XCTAssert(app.keyboards.count > 0, "The keyboard is not shown")
         app.typeText("example.com\n")
 
-//        waitUntilPageLoad()
+        // waitUntilPageLoad()
```

Comment:
Nit: can we remove this instead of leaving it commented?

Reply:
I can update this in the next PR if that is ok with you.

Reply:
sure, no problem